calico-cloud_versioned_docs/version-20-2/release-notes/index.mdx

```diff
@@ -5,6 +5,16 @@ title: Release notes
 
 # Calico Cloud release notes
 
+## December 19, 2024 (version 20.3.1)
+
+### Bug fixes
+
+* Previously, updating Calico Cloud introduced breaking changes for some users who had defined network policy tiers with orders higher than the default value.
+  This was caused by a change to the default tier order from 1,000 to 1,000,000.
```

Comment:
@tmjd I got the 1000 figure from the docs. Is that right?

Reply:
No that's not right, it went from undefined to 1,000,000